src/applications/personalization/profile360/components/PaymentInformationEditModal.jsx

```diff
@@ -0,0 +1,228 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Modal from '@department-of-veterans-affairs/formation-react/Modal';
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+import ErrorableTextInput from '@department-of-veterans-affairs/formation-react/ErrorableTextInput';
+import ErrorableSelect from '@department-of-veterans-affairs/formation-react/ErrorableSelect';
+
+const ACCOUNT_TYPES_OPTIONS = {
+  checking: 'Checking',
+  savings: 'Savings',
+};
+
+class PaymentInformationEditModal extends React.Component {
+  static propTypes = {
+    onClose: PropTypes.func.isRequired,
+    onSubmit: PropTypes.func.isRequired,
+    setPaymentInformationUiState: PropTypes.func.isRequired,
+    paymentInformationUiState: PropTypes.object,
+  };
+
+  static defaultEditModalFields = {
```

Comment:
Why not put this in the initial state of the Redux store?

Reply:
I need to reset the form after closing/opening the modal so I kept these values close by to do that, but if we're in favor of moving validation into the reducer, then it makes sense to me to do that reset in the reducer as well.

Reply:
Added to the initial state